open-api/rest-catalog-open-api.yaml

```diff
@@ -61,6 +61,14 @@ security:
   - OAuth2: [catalog]
   - BearerAuth: []
 
+tags:
```

Comment:
I think that we should add `tables` to clearly call out the parts that are required for table support.

Reply:
I'd like to add to the overall description a paragraph on how tags are used to define capabilities and what the expectations are.

Reply:
For a capability like `tables` I would assume that this is the default capability that each server provides without actually indicating this as part of `capabilities`. 
The reason for this is: A newer client knowning about `capabilities` should still be able to call the namespaces/tables endpoints of an older server that doesn't send back `capabilities`.

Reply:
I guess the question I have is whether a service that supports views must also support tables. Couldn't there be a catalog that only supports views?

Reply:
+1, tables should be a separated capability. However, within tables, there are APIs like emit metrics, register tables, which is not really "common" features that everyone might want. For example, Databricks Unity only offers ListTables and LoadTable, which is actually sufficient for many use cases. This makes me think if it is better to just fully describe every API that is supported...

Reply:
@nastra, looks like this still needs to be addressed.

Reply:
Would we want to have a separate `table-metrics`capability for the metrics endpoint or would we assume that this falls under `tables`? Also would se assume the /config endpoint to have a separate `config` capability or would that also fall under `tables` (since it's an existing capability)?

Reply:
I'd advocate for a different `table-metrics` capability, since I think it is enough of a differentiation for a REST implementation that users should consider, and as @jackye1995 also mentioned it's not really a "common" enough feature that it is implicit. For config, as discussed in the sync I think it's implicit (assumed required if someone implements a REST server), but wonder what others think.

Reply:
I just wanted to mention that metrics reporting via REST can be controlled with the `rest-metrics-reporting-enabled` flags, but I'm ok introducing `table-metrics` if people think we should have that

Reply:
Indeed there is the `rest-metrics-reporting-enabled` flag but I still think it's valuable to introduce `table-metrics` because "enabled" is not the same as "implemented" (e.g. you can have the capability implemented, but disable it momentarily).